test/ModAttachr/modAttachr.ts

```diff
@@ -0,0 +1,36 @@
+import { ModAttachr } from "../../src/ModAttachr";
+
+import { mochaLoader } from "../main";
+
+import { EventNames } from "../../src/EventNames";
+
+import {
+    IMod
+} from "../../src/IModAttachr";
```

Comment:
I prefer keeping lines of local imports all together. No need for a blank line between each of them.